### PR TITLE
[PJLinkDevice] audio/video mute

### PR DESCRIPTION
**Pull request by nils**
As requested on the [community](https://community.openhab.org/t/new-pjlink-binding-for-controlling-multiple-projector-brands-models/56608/58).

### CODE REVIEW

**Comment by nils** [comment]
Tested successfully using the PJLinkTEST4CNT application. Ready to be reviewed :slightly_smiling_face:

**Comment by lolodomo** [comment]
As you add a new channel, please provide upgrade instructions.

**Comment by nils** [comment]
Do you have an example of what needs to be considered for an upgrade when adding a channel?
I thought purely adding shouldn't cause any manual efforts...

**Comment by lolodomo** [comment]
Google helped me
https://www.openhab.org/docs/developer/bindings/thing-xml.html#updating-thing-types

File: bundles/org.openhab.binding.pjlinkdevice/src/main/resources/OH-INF/thing/thing-types.xml [hunk 1]
```diff
@@ -103,6 +104,12 @@
 		<description>Select the video mute status</description>
 	</channel-type>
 
+	<channel-type id="audioAndVideoMute">
```
Comment (lolodomo) [inline bundles/org.openhab.binding.pjlinkdevice/src/main/resources/OH-INF/thing/thing-types.xml:107]: In addition to provide upgrade instructions for the thing, please also regenerate the i18n file to include your new channel: `mvn i18n:generate-default-translations`

**Review by lolodomo — CHANGES_REQUESTED** [review]

**Comment by lsiepel** [comment]
Gentel ping @nils do you need anything to proceed?

File: bundles/org.openhab.binding.pjlinkdevice/src/main/resources/OH-INF/thing/thing-types.xml (hunk 1 above)
Comment (lsiepel) [reply bundles/org.openhab.binding.pjlinkdevice/src/main/resources/OH-INF/thing/thing-types.xml:111]: As these are small changes, i added both updated translation and instructions. Please proceed with the review if possible.

**Review by lolodomo — APPROVED** [review]
LGTM, thank you

**Comment by lolodomo** [comment]
@lsiepel : thank you for finishing the job.

Build is failing.

**Pull request merged**